src/node_options.cc

```diff
@@ -611,6 +611,12 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             "set environment variables from supplied file",
             &EnvironmentOptions::env_file);
   Implies("--env-file", "[has_env_file_string]");
+  AddOption(
+      "--env-file-override-local",
+      "override environment variables set on machine with variables from "
+      "supplied files via --env-file flag or loadEnvFile() process function",
```

Comment:
Is that correct that the flag has influence over `loadEnvFile()`?
```suggestion
      "override environment variables set on machine with variables from "
      "supplied files via --env-file flag",
```